Contribution templates

Roxxas96 opened the pull request:
## Related Issue
Closes #2 

## Overall description of your work
Created a CONTRIBUTING.md to guide people through the conventions to respect in order to contribute to the project & added some templates for Issues & PRs to respect in order to have a clear workflow based on feature branches.

## Task list
- [X] CONTRIBUTING.md
  - [X] Commits
  - [X] Issues
  - [X] Pull Requests
  - [X] Reviews
- [X] Issue & PR templates
  - [X] Bug report Issues
  - [X] Feature request Issues
  - [X] PR template

Roxxas96 committed:
feat: created template for prs & issues

Created templates for PRs and Issues that must be followed by any one who wants to contribute to the project.

Roxxas96 committed:
feat: created the contributing.md

Created the CONTRIBUTING.ms that every one must follow to contribute to this project

RedbeanGit commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,89 @@
+
+The **main** repository [web-bde](https://github.com/Web-BDE/web-bde) is composed of **submodules**, each submodules concern a particular **micro-service** ([front-web-bde](https://github.com/Web-BDE/front-web-bde) for example is the service that distribute static files). Each micro service project has its own Issues/PR and follow **this workflow**.
+
+In each repository yo wil find 3 types of branch:
```
```suggestion
In each repository you will find 3 types of branch:
```

Roxxas96 replied:
Thank you for the typo !

RedbeanGit commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,89 @@
+
+- [x] Write the contributing.md
+
+## When an how to make a Pull Request ?
```
```suggestion
## When and how to make a Pull Request ?
```

Roxxas96 replied:
Thank you !

RedbeanGit reviewed (approved):
Except for a few typos, everything looks good to me

Roxxas96 commented:
[Refacto]

I changed some descriptions about the branches type, added a section for fixup commits and submodules.
I also squashed fixup commis so the PR is ready to merge.
If you want to review the end file @RedbeanGit you can until tomorrow when I'll merge the branch.

Roxxas96 reviewed (commented):
[Review]

I changed some descriptions about the branches type, added a section for fixup commits and submodules.
I also squashed fixup commis so the PR is ready to merge.
If you want to review the end file @RedbeanGit you can until tomorrow when I'll merge the branch.
